### PR TITLE
Fix Whalebird font stack

### LINKED ISSUE #708: Some emojis are rendered incorrect
## How To Reproduce
1. Open New Toot modal
2. Open Character Viewer of macOS
3. Select the woman face-palming and click to inset

## Your Environment
 - OS: MacOS
 - Whalebird Version: 2.5.0

### PR DESCRIPTION
**Pull request by amyspark**
This pull request removes some Japanese and Western language fonts that break emoji composition. I've also removed Lato from the New Toot modal for the same reasons.
Additionally, I've added Bootstrap's font stack as a fallback.
Fixes #708.

### CODE REVIEW

File: src/renderer/App.vue [hunk 1]
```diff
@@ -74,7 +74,7 @@ html, body, #app {
     background-color: #d9e1e8;
   }
 
-  --specified-fonts: 'Noto Sans', 'Noto Sans CJK JP', 'Takaoゴシック', 'Hiragino Kaku Gothic ProN', 'ヒラギノ角ゴ ProN W3', Meiryo, メイリオ, Osaka, 'MS PGothic', arial, helvetica, sans-serif;
+  --specified-fonts: 'Noto Sans', 'Noto Sans CJK JP', 'Takaoゴシック', 'ヒラギノ角ゴ ProN W3', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Helvetica Neue', 'Apple Color Emoji', 'Segoe UI Emoji','Segoe UI Symbol';
```
Comment (h3poteto) [inline src/renderer/App.vue]: __[stylelint]__ <sub>posted by Sider</sub>

Expected single space after "," in a single-line list (value-list-comma-space-after)

File: src/renderer/components/TimelineSpace/Modals/NewToot/Status.vue [hunk 2]
```diff
@@ -319,7 +319,6 @@ export default {
     resize: none;
     height: 120px;
     transition: border-color 0.2s cubic-bezier(0.645, 0.045, 9.355, 1);
-    font-family: 'Lato', sans-serif;
```
Comment (h3poteto) [inline src/renderer/components/TimelineSpace/Modals/NewToot/Status.vue]: Hmm... The reason of Lato is a bug of Noto Sans.
Noto Sans could not render backticks properly, so I have set Lato. 
https://github.com/h3poteto/whalebird-desktop/issues/382

Comment (amyspark) [reply src/renderer/components/TimelineSpace/Modals/NewToot/Status.vue]: Found the issue, but I can't reproduce it. My Noto Sans version is 2.0 (courtesy of TeX Live). Perhaps it's been updated?

Comment (h3poteto) [reply src/renderer/components/TimelineSpace/Modals/NewToot/Status.vue]: I still reproduce it on my computer.
screenshot from 2018-11-21 21-40-43

Comment (amyspark) [reply src/renderer/components/TimelineSpace/Modals/NewToot/Status.vue]: I can reproduce it too *only with the embedded version in Whalebird*.
</img>

Noto Sans 2.x works correctly.
</img>

Comment (h3poteto) [reply src/renderer/components/TimelineSpace/Modals/NewToot/Status.vue]: Oh, I see, thank you.
I'll update Noto Sans in Whalebird.

**Pull request merged**